src/utils.jl

```diff
@@ -0,0 +1,22 @@
+macro src()
```

Comment:
Looks like another accidental rebase reversion of https://github.com/JuliaWeb/HTTP.jl/commit/5ba1fd39a720c875bbb780f5d3926a70de5aec55